bcda/web/api.go

```diff
@@ -594,6 +594,12 @@ func healthCheck(w http.ResponseWriter, r *http.Request) {
 func getAuthInfo(w http.ResponseWriter, r *http.Request) {
 	respMap := make(map[string]string)
 	respMap["auth_provider"] = auth.GetProviderName()
+	version, err := auth.GetProvider().GetVersion()
+	if err == nil {
+		respMap["version"] = version
+	} else {
+		respMap["error message"] = err.Error()
```

Comment:
Sorry Pat, I was more thinking of doing something like the following:

```
http.Error(w, err.Error(), http.StatusInternalServerError)
```

What do you think? Might be more clear to return 500 over returning 200 with the error in the body.
```

Reply:
@mtrang1263 i'm not sure. this is at the end of the day a status page and users are gonna want to know what the set  "auth_provider" is. i don't think throwing a 500 error for this is the right move

Reply:
Good point! Better to render something for the status page versus an empty page/empty response.